### DISCUSSION TITLE
Welcome to esp-homekit-devices Discussions!

### DISCUSSION THREAD

**Discussion by RavenSystem** [RavenSystem/esp-homekit-devices#1493] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community, leaving issues section only to report bugs. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by peros550** [comment]
HAA: the best project ever!

**Comment by Daveblanche** [comment]
Hallo!
I’m Dave.
I love Raven’s HAA.
I’m a hobby guy who moved into a new house just before covid and used the time to learn about HAA, adding lights and fans to all parts of it.
There are, I think 23 installed  Sonoff units around the house.

It has been fun and I have learned lots.

That Json stuff still gets me. More time required.

**Comment by TomBBPL** [comment]
Hi!
5 years of searching for the one and only best system. (custom codes, EspEasy, Tasmota, Blink, Domoticz, HA, Homebridge, ...).
I recently found HAA and that's it! 
My search is over, now it's time for a great adventure with the brilliant HAA project!

Thanks guys for your work, you are great!
